### ISSUE TITLE
Implement `console.table()` method

### ISSUE THREAD

**Issue by HalidOdat** [boa-dev/boa#3806]
Specification: <https://console.spec.whatwg.org/#table>
MDN Documentation: <https://developer.mozilla.org/en-US/docs/Web/API/console/table_static>

**Comment by TomasMatarazzo** [comment]
can i work on this? thx

**Comment by HalidOdat** [comment]
Sure, I'll assign you :)

**Comment by hunterchen7** [comment]
Hi, I haven't seen any movement on this for a bit, would it be ok if I gave it a go?

**Comment by andrewbovbel** [comment]
I'm going to start this. Is that okay?

**Comment by sumitparakh** [comment]
@jedel1043

Hi Jose, it's been 6 months and if no one else is working on this then can I please take this up? Thanks

**Comment by jedel1043** [comment]
@sumitparakh Sure! I'll assign you.

**Comment by snark87** [comment]
@jedel1043 Hi Jose! As discussed in PR, I'd like to help with this issue.

**Comment by jedel1043** [comment]
@snark87 Of course! I'll assign it to you.

**Comment by TimoRasenbergTU** [comment]
Is @snark87 still actively working on this issue?

**Comment by jedel1043** [comment]
@TimoRasenbergTU Nope, I don't think we heard back from them. Do you wanna work on this?

**Comment by nae3x** [comment]
Hello, I would like to work on this issue. Please assign me, thank you.

**Comment by ParthMozarkar** [comment]
Hi, I’m currently setting up Boa locally on Windows and would like to work on implementing `console.table()` as my first contribution.

Could you please guide me on where in the codebase the console methods are currently implemented so I can get started?

**Comment by plasmaDestroyer** [comment]
Hey @jedel1043! I would like to work on this issue.

**Comment by jedel1043** [comment]
Since this issue has been kind of hijacked multiple times for some reason, just open a _finished_ PR (not drafts) and we'll accept the first one that's at least _finished_

**Comment by ParthMozarkar** [comment]
hey, could you please review once the pr raised ?

**Comment by official-Jatin** [comment]
Hi! I'd like to work on this. I've already explored the codebase and understand the console implementation pattern in core/runtime/src/console/mod.rs. I plan to implement console.table() following the [WHATWG spec](https://console.spec.whatwg.org/#table). Would love any guidance from maintainers!

**Comment by jedel1043** [comment]
This issue is free to be picked up again

**Comment by hunterchen7** [comment]
gonna give this a shot with comfy table :)

can i open a PR if/when i think I've made some good progress?

**Comment by jedel1043** [comment]
Yep, go for it

**Comment by hunterchen7** [comment]
> Yep, go for it

Hi, I've opened a PR, could I get a review on it when you have some time? :)

https://github.com/boa-dev/boa/pull/5218

**Comment by hunterchen7** [comment]
hi @jedel1043 just wanted to bump this thread
